network_manager/__init__.py

```diff
@@ -10,19 +10,18 @@ def main(global_config, **settings):
     Base.metadata.bind = engine
 
     config = Configurator(settings=settings)
+
     config.include("pyramid_chameleon")
 
-    # Add a route / path called node_all at /node/update
-    # This is the 'admin' page
-    # This is for displaying all entries in the database at the top of the page and a blank form
-    config.add_route("node_all", "/node/update")
+    config.add_route("node_edit", "/node/edit", request_method=("POST", "GET"))
```

Comment:
Updated the routes so that we are implementing the ones stated in https://github.com/DACCS-Climate/roadmap/issues/22.